components/consentRequestForm/index.jsx

```diff
@@ -118,6 +116,8 @@ export default function ConsentRequestForm({ agentDetails, agentWebId }) {
           requestor,
           access: selectedAccess.accessModes,
           resources: selectedResources,
+          expirationDate: new Date(selectedDate),
```

Comment:
You'll want to conditionally set this, I think, so perhaps move this object up into a `const approvalOverrides` object, and then `if (selectedDate !== null) { approvalOverrides.expirationDate = new Date(selectedDate); }`

Reply:
I guess it depends on what "Forever" means, does it mean Infinity, or does it mean "whatever the requestor wanted originally"